src/main/features/response/views/claim-details.njk

```diff
@@ -16,7 +16,8 @@
       {{ amountBreakdownTable('View amount breakdown', claim, interestData) }}
       <p><span class="bold-small">{{ t('Reason for claim') }}:</span></p>
       <p>{{ claim.claimData.reason }}</p>
-
+      <p><span class="bold-small">{{ t('Download the claim') }}</span></p>
+      <p><span class="block"><a href="{{ ClaimPaths.receiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download claim') }}</a> ({{ t('PDF') }})</span></p>
```

Comment:
same as above, can these spans be moved into the p's a the classes applied there if still needed? I don't think they will be needed though

Reply:
Done